sound/soc/intel/boards/sof_sdw_rt_amp.c

```diff
@@ -169,8 +176,10 @@ static const struct snd_soc_dapm_route *get_codec_name_and_route(struct snd_soc_
 		return rt1308_map;
 	else if (strcmp(codec_name, "rt1316") == 0)
 		return rt1316_map;
-	else
+	else if (strcmp(codec_name, "rt1318") == 0)
 		return rt1318_map;
+	else
+		return rt1320_map;
```

Comment:
@bardliao this is going to conflict with @vijendarmukunda 's refactoring. Once of the two needs to go first...

Reply:
@bardliao : PR#5068 is rebase version (which has been rebased 7 times already). Could you please help to review PR#5068? Let PR#5068 go first after clearing code review. Because as this patch series is depends on Intel Code, every time when ever minor code changes done in Intel code, I was forced to rebase.  As it's initial version Machine driver for AMD platforms. We want to work on incremental way.

Reply:
> @bardliao : PR#5068 is rebase version (which has been rebased 7 times already). Could you please help to review PR#5068? Let PR#5068 go first after clearing code review. Because as this patch series is depends on Intel Code, every time when ever minor code changes done in Intel code, I was forced to rebase. As it's initial version Machine driver for AMD platforms. We want to work on incremental way.

Sure, will review it today